README.jp.md

```diff
@@ -0,0 +1,803 @@
+を必要としません。
+このアプリケーションは　_GNU/Linux_, _Windows_ そして _macOS_ 上で動作します。
+
+![screenshot](assets/screenshot-debian-600.jpg)
+
+以下に焦点を当てています:
+
+ - **軽量** (ネイティブ、デバイス画面表示のみ)
+ - **パフォーマンス**　(30~60fps)
+ - **クオリティ**　(1920x1080以上)
+ - **低遅延**　([35~70ms][lowlatency])
+ - **短い起動時間**　(初回画像を1秒以内に表示)
+ - **非侵入型**　(デバイスに何もインストールされない)
+
+[lowlatency]: https://github.com/Genymobile/scrcpy/pull/646
+
+
+## 必要要件
+
+AndroidデバイスはAPI21(Android 5.0)以上。
+
+Androidデバイスで[adbデバッグが有効][enable-adb]であること。
+
+[enable-adb]: https://developer.android.com/studio/command-line/adb.html#Enabling
+
+一部ののAndroidデバイスでは、キーボードとマウスを使用して制御する[追加オプション][control]
+を有効にする必要がある。
+
+[control]: https://github.com/Genymobile/scrcpy/issues/70#issuecomment-373286323
+
+
+## アプリの取得
+
+<a href="https://repology.org/project/scrcpy/versions"><img src="https://repology.org/badge/vertical-allrepos/scrcpy.svg" alt="Packaging status" align="right"></a>
+
+### Linux
+
+Debian (_testing_ と _sid_) とUbuntu(20.04):
+
+```
+apt install scrcpy
+```
+
+[Snap]パッケージが利用可能: [`scrcpy`][snap-link]
+
+[snap-link]: https://ja.wikipedia.org/wiki/Snappy
+
+[snap]: https://en.wikipedia.org/wiki/Snappy_(package_manager)
+
+Fedora用[COPR]パッケージが利用可能: [`scrcpy`][copr-link]
+
+[COPR]: https://fedoraproject.org/wiki/Category:Copr
+[copr-link]: https://copr.fedorainfracloud.org/coprs/zeno/scrcpy/
+
+Arch Linux用[AUR]パッケージが利用可能: [`scrcpy`][aur-link]
+
+[AUR]: https://wiki.archlinux.org/index.php/Arch_User_Repository
+[aur-link]: https://aur.archlinux.org/packages/scrcpy/
+
+Gentoo用[Ebuild]が利用可能: [`scrcpy`][ebuild-link]
+
+[Ebuild]: https://wiki.gentoo.org/wiki/Ebuild
+[ebuild-link]: https://github.com/maggu2810/maggu2810-overlay/tree/master/app-mobilephone/scrcpy
+
+[自分でビルド][BUILD]も可能（心配しないでください、それほど難しくはありません。）
+
+
+### Windows
+
+Windowsでは簡単に、（`adb`を含む）すべての依存関係を構築済みのアーカイブを利用可能です。
+
+ - [`scrcpy-win64-v1.17.zip`][direct-win64]  
+   _(SHA-256: 8b9e57993c707367ed10ebfe0e1ef563c7a29d9af4a355cd8b6a52a317c73eea)_
+
+[direct-win64]: https://github.com/Genymobile/scrcpy/releases/download/v1.17/scrcpy-win64-v1.17.zip
+
+[Chocolatey]でも利用可能です:
+
+[Chocolatey]: https://chocolatey.org/
+
+```bash
+choco install scrcpy
+choco install adb    # まだ入手していない場合
+```
+
+[Scoop]でも利用可能です:
+
+```bash
+scoop install scrcpy
+scoop install adb    # まだ入手していない場合
+```
+
+[Scoop]: https://scoop.sh
+
+また、[アプリケーションをビルド][BUILD]することも可能です。
+
+### macOS
+
+アプリケーションは[Homebrew]で利用可能です。ただインストールするだけです。
+
+[Homebrew]: https://brew.sh/
+
+```bash
+brew install scrcpy
+```
+
+`PATH`から`adb`へのアクセスが必要です。もしまだ持っていない場合:
+
+```bash
+# Homebrew >= 2.6.0
+brew install --cask android-platform-tools
+
+# Homebrew < 2.6.0
+brew cask install android-platform-tools
+```
+
+また、[アプリケーションをビルド][BUILD]することも可能です。
+
+
+## 実行
+
+Androidデバイスを接続し、実行:
+
+```bash
+scrcpy
+```
+
+次のコマンドでリストされるコマンドライン引数も受け付けます:
+
+```bash
+scrcpy --help
+```
+
+## 機能
+
+### キャプチャ構成
+
+#### サイズ削減
+
+Androidデバイスを低解像度でミラーリングする場合、パフォーマンス向上に便利な
+場合があります。
+
+幅と高さをある値に制限するには(例：1024):
```

Comment:
```diff
- 幅と高さをある値に制限するには(例：1024):
+ 幅と高さをある値(例：1024)に制限するには:
```

In fact `--max-size` will limit the larger one of width and height to the given value ("**both** the width and height"). Need more clarification.